editions/tw5.com/tiddlers/nodejs/Using TiddlyWiki on Node.js.tid

```diff
@@ -1,18 +1,18 @@
 created: 20131219100520659
-modified: 20200523092939199
+modified: 20220613172345723
 tags: [[TiddlyWiki on Node.js]]
 title: Using TiddlyWiki on Node.js
 type: text/vnd.tiddlywiki
 
-TiddlyWiki5 includes a set of [[Commands]] for use on the command line to perform an extensive set of operations based on TiddlyWikiFolders, TiddlerFiles and TiddlyWikiFiles.
+~TiddlyWiki5 includes a set of commands for use on the command line to perform an extensive set of operations based on TiddlyWikiFolders, TiddlerFiles.
```

Comment:
Why remove the link to commands? better to replace it with `[[commands|Commands]]`

Reply:
If users follow the link that early, imo it is a distruction. They didn't even read the basics about how to use the CLI. There is a specific link at the end of the tiddler. IMO that's the right place to follow a link to get more info